.github/workflows/push.yml

```diff
@@ -7,8 +7,10 @@ jobs:
     strategy:
       matrix:
         ruby:
-          - '3.0'
-          - '3.3'
+          - 3.0
```

Comment:
This one must be quoted, otherwise it's read as "3" and will use the latest Ruby 3.

Reply:
Thanks for catching that and reminding me why I hate YAML 😁